static/app/components/tooltip.tsx

```diff
@@ -229,20 +148,21 @@ function Tooltip({
     []
   );
 
-  // When not using a global portal we need to be sure to cleanup tooltip
-  // portals from the DOM
-  useEffect(
-    () => () => void !useGlobalPortal && document.body.removeChild(tooltipPortal),
-    [useGlobalPortal, tooltipPortal]
-  );
-
-  // Register functionality for visual acceptance tests
-  useTooltipInAcceptanceTests({
-    tooltipId,
-    setOpen,
-    setUseGlobalPortal,
-    ignore: disabled || disableForVisualTest,
-  });
+  // Tracks the triggering element
+  const triggerRef = useRef<HTMLElement | null>(null);
+  const modifiers: PopperJS.Modifiers = useMemo(() => {
+    return {
+      hide: {enabled: false},
+      preventOverflow: {
+        padding: 10,
+        enabled: true,
+        boundariesElement: 'viewport',
+      },
+      applyStyle: {
+        gpuAcceleration: true,
+      },
+    };
+  }, []);
```

Comment:
Why do we memoize the modifiers?

Reply:
So we  have a stable object reference, avoid potentially unnecessary rerenders